*** Starting uWSGI 2.0.10 (64bit) on [Sat May 23 14:15:45 2015] ***
compiled with version: 4.8.2 20140120 (Red Hat 4.8.2-16) on 23 May 2015 13:22:55
os: Linux-3.14.42-31.38.amzn1.x86_64 #1 SMP Wed May 13 20:33:05 UTC 2015
nodename: ip-172-31-21-203
machine: x86_64
clock source: unix
detected number of CPU cores: 1
current working directory: /home/ec2-user/projects/chaos.trade
detected binary path: /usr/local/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
*** WARNING: you are running uWSGI without its master process manager ***
your processes number limit is 7826
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address /home/ec2-user/projects/chaos.trade/chaos_uwsgi.sock fd 3
Python version: 2.7.9 (default, Apr  1 2015, 18:18:03)  [GCC 4.8.2 20140120 (Red Hat 4.8.2-16)]
Set PythonHome to /home/ec2-user/projects/chaos.trade
ImportError: No module named site
*** Starting uWSGI 2.0.10 (64bit) on [Sat May 23 14:22:28 2015] ***
compiled with version: 4.8.2 20140120 (Red Hat 4.8.2-16) on 23 May 2015 13:22:55
os: Linux-3.14.42-31.38.amzn1.x86_64 #1 SMP Wed May 13 20:33:05 UTC 2015
nodename: ip-172-31-21-203
machine: x86_64
clock source: unix
detected number of CPU cores: 1
current working directory: /home/ec2-user/projects/chaos.trade
detected binary path: /usr/local/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
*** WARNING: you are running uWSGI without its master process manager ***
your processes number limit is 7826
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address /home/ec2-user/projects/chaos.trade/chaos_uwsgi.sock fd 3
Python version: 2.7.9 (default, Apr  1 2015, 18:18:03)  [GCC 4.8.2 20140120 (Red Hat 4.8.2-16)]
Set PythonHome to /home/ec2-user/projects/chaos.trade
ImportError: No module named site
*** Starting uWSGI 2.0.10 (64bit) on [Sat May 23 14:26:42 2015] ***
compiled with version: 4.8.2 20140120 (Red Hat 4.8.2-16) on 23 May 2015 13:22:55
os: Linux-3.14.42-31.38.amzn1.x86_64 #1 SMP Wed May 13 20:33:05 UTC 2015
nodename: ip-172-31-21-203
machine: x86_64
clock source: unix
detected number of CPU cores: 1
current working directory: /home/ec2-user/projects/chaos.trade
detected binary path: /usr/local/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
*** WARNING: you are running uWSGI without its master process manager ***
your processes number limit is 7826
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address /home/ec2-user/projects/chaos.trade/chaos_uwsgi.sock fd 3
Python version: 2.7.9 (default, Apr  1 2015, 18:18:03)  [GCC 4.8.2 20140120 (Red Hat 4.8.2-16)]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0xc2cd70
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 72768 bytes (71 KB) for 1 cores
*** Operational MODE: single process ***
added /home/ec2-user/projects/chaos.trade/ to pythonpath.
WSGI app 0 (mountpoint='') ready in 0 seconds on interpreter 0xc2cd70 pid: 8038 (default app)
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI worker 1 (and the only) (pid: 8038, cores: 1)
*** Starting uWSGI 2.0.10 (64bit) on [Sat May 23 14:31:06 2015] ***
compiled with version: 4.8.2 20140120 (Red Hat 4.8.2-16) on 23 May 2015 13:22:55
os: Linux-3.14.42-31.38.amzn1.x86_64 #1 SMP Wed May 13 20:33:05 UTC 2015
nodename: ip-172-31-21-203
machine: x86_64
clock source: unix
detected number of CPU cores: 1
current working directory: /home/ec2-user/projects/chaos.trade
detected binary path: /usr/local/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
*** WARNING: you are running uWSGI without its master process manager ***
your processes number limit is 7826
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address /home/ec2-user/projects/chaos.trade/chaos_uwsgi.sock fd 3
Python version: 2.7.9 (default, Apr  1 2015, 18:18:03)  [GCC 4.8.2 20140120 (Red Hat 4.8.2-16)]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x7eae10
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 72768 bytes (71 KB) for 1 cores
*** Operational MODE: single process ***
added /home/ec2-user/projects/chaos.trade/ to pythonpath.
Traceback (most recent call last):
  File "/home/ec2-user/projects/chaos.trade/chaos.py", line 6, in <module>
    from flask import Flask
ImportError: No module named flask
unable to load app 0 (mountpoint='') (callable not found or import error)
*** no app loaded. going in full dynamic mode ***
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI worker 1 (and the only) (pid: 8042, cores: 1)
*** Starting uWSGI 2.0.10 (64bit) on [Sat May 23 14:37:57 2015] ***
compiled with version: 4.8.2 20140120 (Red Hat 4.8.2-16) on 23 May 2015 13:22:55
os: Linux-3.14.42-31.38.amzn1.x86_64 #1 SMP Wed May 13 20:33:05 UTC 2015
nodename: ip-172-31-21-203
machine: x86_64
clock source: unix
detected number of CPU cores: 1
current working directory: /home/ec2-user/projects/chaos.trade
detected binary path: /usr/local/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
chdir() to /home/ec2-user/projects/chaos.trade
*** WARNING: you are running uWSGI without its master process manager ***
your processes number limit is 7826
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address /home/ec2-user/projects/chaos.trade/chaos_uwsgi.sock fd 3
Python version: 2.7.9 (default, Apr  1 2015, 18:18:03)  [GCC 4.8.2 20140120 (Red Hat 4.8.2-16)]
Set PythonHome to /home/ec2-user/projects/chaos.trade
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x2601e60
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 72768 bytes (71 KB) for 1 cores
*** Operational MODE: single process ***
added /home/ec2-user/projects/chaos.trade/ to pythonpath.
Traceback (most recent call last):
  File "/home/ec2-user/projects/chaos.trade/chaos.py", line 6, in <module>
    from flask import Flask
ImportError: No module named flask
unable to load app 0 (mountpoint='') (callable not found or import error)
*** no app loaded. going in full dynamic mode ***
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI worker 1 (and the only) (pid: 8051, cores: 1)
*** Starting uWSGI 2.0.10 (64bit) on [Sat May 23 14:41:27 2015] ***
compiled with version: 4.8.2 20140120 (Red Hat 4.8.2-16) on 23 May 2015 13:22:55
os: Linux-3.14.42-31.38.amzn1.x86_64 #1 SMP Wed May 13 20:33:05 UTC 2015
nodename: ip-172-31-21-203
machine: x86_64
clock source: unix
detected number of CPU cores: 1
current working directory: /home/ec2-user/projects/chaos.trade
detected binary path: /usr/local/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
chdir() to /home/ec2-user/projects/chaos.trade
*** WARNING: you are running uWSGI without its master process manager ***
your processes number limit is 7826
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address /home/ec2-user/projects/chaos.trade/chaos_uwsgi.sock fd 3
Python version: 2.7.9 (default, Apr  1 2015, 18:18:03)  [GCC 4.8.2 20140120 (Red Hat 4.8.2-16)]
Set PythonHome to /usr/local/lib/python2.7/site-packages
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0xb9bec0
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 72768 bytes (71 KB) for 1 cores
*** Operational MODE: single process ***
added /home/ec2-user/projects/chaos.trade/ to pythonpath.
Traceback (most recent call last):
  File "/home/ec2-user/projects/chaos.trade/chaos.py", line 6, in <module>
    from flask import Flask
ImportError: No module named flask
unable to load app 0 (mountpoint='') (callable not found or import error)
*** no app loaded. going in full dynamic mode ***
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI worker 1 (and the only) (pid: 8059, cores: 1)
*** Starting uWSGI 2.0.10 (64bit) on [Sat May 23 14:48:36 2015] ***
compiled with version: 4.8.2 20140120 (Red Hat 4.8.2-16) on 23 May 2015 13:22:55
os: Linux-3.14.42-31.38.amzn1.x86_64 #1 SMP Wed May 13 20:33:05 UTC 2015
nodename: ip-172-31-21-203
machine: x86_64
clock source: unix
detected number of CPU cores: 1
current working directory: /home/ec2-user/projects/chaos.trade
detected binary path: /usr/local/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
chdir() to /home/ec2-user/projects/chaos.trade
*** WARNING: you are running uWSGI without its master process manager ***
your processes number limit is 7826
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address /home/ec2-user/projects/chaos.trade/chaos_uwsgi.sock fd 3
Python version: 2.7.9 (default, Apr  1 2015, 18:18:03)  [GCC 4.8.2 20140120 (Red Hat 4.8.2-16)]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x1a66e80
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 72768 bytes (71 KB) for 1 cores
*** Operational MODE: single process ***
added /home/ec2-user/projects/chaos.trade;/usr/local/lib/python2.7/site-packages to pythonpath.
Traceback (most recent call last):
  File "./chaos.py", line 6, in <module>
    from flask import Flask
ImportError: No module named flask
unable to load app 0 (mountpoint='') (callable not found or import error)
*** no app loaded. going in full dynamic mode ***
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI worker 1 (and the only) (pid: 8095, cores: 1)
*** Starting uWSGI 2.0.10 (64bit) on [Sat May 23 14:53:12 2015] ***
compiled with version: 4.8.2 20140120 (Red Hat 4.8.2-16) on 23 May 2015 13:22:55
os: Linux-3.14.42-31.38.amzn1.x86_64 #1 SMP Wed May 13 20:33:05 UTC 2015
nodename: ip-172-31-21-203
machine: x86_64
clock source: unix
detected number of CPU cores: 1
current working directory: /home/ec2-user/projects/chaos.trade
detected binary path: /usr/local/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
chdir() to /home/ec2-user/projects/chaos.trade
*** WARNING: you are running uWSGI without its master process manager ***
your processes number limit is 7826
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address /home/ec2-user/projects/chaos.trade/chaos_uwsgi.sock fd 3
Python version: 2.7.9 (default, Apr  1 2015, 18:18:03)  [GCC 4.8.2 20140120 (Red Hat 4.8.2-16)]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x7cde10
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 72768 bytes (71 KB) for 1 cores
*** Operational MODE: single process ***
added /home/ec2-user/projects/chaos.trade;/usr/local/lib/python2.7/site-packages to pythonpath.
Traceback (most recent call last):
  File "./chaos.py", line 6, in <module>
    from flask import Flask
ImportError: No module named flask
unable to load app 0 (mountpoint='') (callable not found or import error)
*** no app loaded. going in full dynamic mode ***
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI worker 1 (and the only) (pid: 8102, cores: 1)
*** Starting uWSGI 2.0.10 (64bit) on [Sat May 23 14:56:59 2015] ***
compiled with version: 4.8.2 20140120 (Red Hat 4.8.2-16) on 23 May 2015 13:22:55
os: Linux-3.14.42-31.38.amzn1.x86_64 #1 SMP Wed May 13 20:33:05 UTC 2015
nodename: ip-172-31-21-203
machine: x86_64
clock source: unix
detected number of CPU cores: 1
current working directory: /home/ec2-user/projects/chaos.trade
detected binary path: /usr/local/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
chdir() to /home/ec2-user/projects/chaos.trade
*** WARNING: you are running uWSGI without its master process manager ***
your processes number limit is 7826
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address /home/ec2-user/projects/chaos.trade/chaos_uwsgi.sock fd 3
Python version: 2.7.9 (default, Apr  1 2015, 18:18:03)  [GCC 4.8.2 20140120 (Red Hat 4.8.2-16)]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x1a2fe10
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 72768 bytes (71 KB) for 1 cores
*** Operational MODE: single process ***
added /home/ec2-user/projects/chaos.trade;/usr/local/lib/python2.7/site-packages to pythonpath.
Traceback (most recent call last):
  File "./chaos.py", line 6, in <module>
    from flask import Flask
ImportError: No module named flask
unable to load app 0 (mountpoint='') (callable not found or import error)
*** no app loaded. going in full dynamic mode ***
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI worker 1 (and the only) (pid: 8123, cores: 1)
*** Starting uWSGI 2.0.10 (64bit) on [Sat May 23 15:07:16 2015] ***
compiled with version: 4.8.2 20140120 (Red Hat 4.8.2-16) on 23 May 2015 13:22:55
os: Linux-3.14.42-31.38.amzn1.x86_64 #1 SMP Wed May 13 20:33:05 UTC 2015
nodename: ip-172-31-21-203
machine: x86_64
clock source: unix
detected number of CPU cores: 1
current working directory: /home/ec2-user/projects/chaos.trade
detected binary path: /usr/local/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
chdir() to /home/ec2-user/projects/chaos.trade
*** WARNING: you are running uWSGI without its master process manager ***
your processes number limit is 7826
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address /home/ec2-user/projects/chaos.trade/chaos_uwsgi.sock fd 3
Python version: 2.7.9 (default, Apr  1 2015, 18:18:03)  [GCC 4.8.2 20140120 (Red Hat 4.8.2-16)]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x1428e10
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 72768 bytes (71 KB) for 1 cores
*** Operational MODE: single process ***
added /home/ec2-user/projects/chaos.trade:/usr/local/lib/python2.7/site-packages to pythonpath.
Traceback (most recent call last):
  File "./chaos.py", line 6, in <module>
    from flask import Flask
ImportError: No module named flask
unable to load app 0 (mountpoint='') (callable not found or import error)
*** no app loaded. going in full dynamic mode ***
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI worker 1 (and the only) (pid: 8163, cores: 1)
*** Starting uWSGI 2.0.10 (64bit) on [Sat May 23 15:09:58 2015] ***
compiled with version: 4.8.2 20140120 (Red Hat 4.8.2-16) on 23 May 2015 13:22:55
os: Linux-3.14.42-31.38.amzn1.x86_64 #1 SMP Wed May 13 20:33:05 UTC 2015
nodename: ip-172-31-21-203
machine: x86_64
clock source: unix
detected number of CPU cores: 1
current working directory: /home/ec2-user/projects/chaos.trade
detected binary path: /usr/local/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
chdir() to /home/ec2-user/projects/chaos.trade
*** WARNING: you are running uWSGI without its master process manager ***
your processes number limit is 7826
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address /home/ec2-user/projects/chaos.trade/chaos_uwsgi.sock fd 3
Python version: 2.7.9 (default, Apr  1 2015, 18:18:03)  [GCC 4.8.2 20140120 (Red Hat 4.8.2-16)]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x2453f70
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 72768 bytes (71 KB) for 1 cores
*** Operational MODE: single process ***
added /home/ec2-user/projects/chaos.trade/ to pythonpath.
added /home/ec2-user/projects/chaos.trade:/usr/local/lib/python2.7/site-packages to pythonpath.
Traceback (most recent call last):
  File "/home/ec2-user/projects/chaos.trade/chaos.py", line 6, in <module>
    from flask import Flask
ImportError: No module named flask
unable to load app 0 (mountpoint='') (callable not found or import error)
*** no app loaded. going in full dynamic mode ***
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI worker 1 (and the only) (pid: 8196, cores: 1)
*** Starting uWSGI 2.0.10 (64bit) on [Sat May 23 15:10:14 2015] ***
compiled with version: 4.8.2 20140120 (Red Hat 4.8.2-16) on 23 May 2015 13:22:55
os: Linux-3.14.42-31.38.amzn1.x86_64 #1 SMP Wed May 13 20:33:05 UTC 2015
nodename: ip-172-31-21-203
machine: x86_64
clock source: unix
detected number of CPU cores: 1
current working directory: /home/ec2-user/projects/chaos.trade
detected binary path: /usr/local/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
chdir() to /home/ec2-user/projects/chaos.trade
*** WARNING: you are running uWSGI without its master process manager ***
your processes number limit is 7826
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address /home/ec2-user/projects/chaos.trade/chaos_uwsgi.sock fd 3
Python version: 2.7.9 (default, Apr  1 2015, 18:18:03)  [GCC 4.8.2 20140120 (Red Hat 4.8.2-16)]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x22ebf70
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 72768 bytes (71 KB) for 1 cores
*** Operational MODE: single process ***
added /home/ec2-user/projects/chaos.trade/ to pythonpath.
added /home/ec2-user/projects/chaos.trade:/usr/local/lib/python2.7/site-packages to pythonpath.
Traceback (most recent call last):
  File "/home/ec2-user/projects/chaos.trade/chaos.py", line 6, in <module>
    from flask import Flask
ImportError: No module named flask
unable to load app 0 (mountpoint='') (callable not found or import error)
*** no app loaded. going in full dynamic mode ***
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI worker 1 (and the only) (pid: 8200, cores: 1)
*** Starting uWSGI 2.0.10 (64bit) on [Sat May 23 15:13:42 2015] ***
compiled with version: 4.8.2 20140120 (Red Hat 4.8.2-16) on 23 May 2015 13:22:55
os: Linux-3.14.42-31.38.amzn1.x86_64 #1 SMP Wed May 13 20:33:05 UTC 2015
nodename: ip-172-31-21-203
machine: x86_64
clock source: unix
detected number of CPU cores: 1
current working directory: /home/ec2-user/projects/chaos.trade
detected binary path: /usr/local/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
chdir() to /home/ec2-user/projects/chaos.trade
*** WARNING: you are running uWSGI without its master process manager ***
your processes number limit is 7826
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address /home/ec2-user/projects/chaos.trade/chaos_uwsgi.sock fd 3
Python version: 2.7.9 (default, Apr  1 2015, 18:18:03)  [GCC 4.8.2 20140120 (Red Hat 4.8.2-16)]
Set PythonHome to /usr/local/lib/python2.7/site-packages
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x1490fa0
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 72768 bytes (71 KB) for 1 cores
*** Operational MODE: single process ***
added /home/ec2-user/projects/chaos.trade/ to pythonpath.
added /usr/local/lib/python2.7/site-packages/ to pythonpath.
Traceback (most recent call last):
  File "/home/ec2-user/projects/chaos.trade/chaos.py", line 6, in <module>
    from flask import Flask
  File "/usr/local/lib/python2.7/site-packages/flask/__init__.py", line 17, in <module>
    from werkzeug.exceptions import abort
  File "/usr/local/lib/python2.7/site-packages/werkzeug/__init__.py", line 17, in <module>
    from types import ModuleType
ImportError: No module named types
unable to load app 0 (mountpoint='') (callable not found or import error)
*** no app loaded. going in full dynamic mode ***
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI worker 1 (and the only) (pid: 8251, cores: 1)
*** Starting uWSGI 2.0.10 (64bit) on [Sat May 23 15:21:25 2015] ***
compiled with version: 4.8.2 20140120 (Red Hat 4.8.2-16) on 23 May 2015 13:22:55
os: Linux-3.14.42-31.38.amzn1.x86_64 #1 SMP Wed May 13 20:33:05 UTC 2015
nodename: ip-172-31-21-203
machine: x86_64
clock source: unix
detected number of CPU cores: 1
current working directory: /home/ec2-user/projects/chaos.trade
detected binary path: /usr/local/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
chdir() to /home/ec2-user/projects/chaos.trade
*** WARNING: you are running uWSGI without its master process manager ***
your processes number limit is 7826
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address /home/ec2-user/projects/chaos.trade/chaos_uwsgi.sock fd 3
Python version: 2.7.9 (default, Apr  1 2015, 18:18:03)  [GCC 4.8.2 20140120 (Red Hat 4.8.2-16)]
Set PythonHome to /usr/local/lib/python2.7/site-packages
ImportError: No module named site
*** Starting uWSGI 2.0.10 (64bit) on [Sat May 23 15:24:02 2015] ***
compiled with version: 4.8.2 20140120 (Red Hat 4.8.2-16) on 23 May 2015 13:22:55
os: Linux-3.14.42-31.38.amzn1.x86_64 #1 SMP Wed May 13 20:33:05 UTC 2015
nodename: ip-172-31-21-203
machine: x86_64
clock source: unix
detected number of CPU cores: 1
current working directory: /home/ec2-user/projects/chaos.trade
detected binary path: /usr/local/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
chdir() to /home/ec2-user/projects/chaos.trade
*** WARNING: you are running uWSGI without its master process manager ***
your processes number limit is 7826
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address /home/ec2-user/projects/chaos.trade/chaos_uwsgi.sock fd 3
Python version: 2.7.9 (default, Apr  1 2015, 18:18:03)  [GCC 4.8.2 20140120 (Red Hat 4.8.2-16)]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x2465e40
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 72768 bytes (71 KB) for 1 cores
*** Operational MODE: single process ***
added /home/ec2-user/projects/chaos.trade:/usr/local/lib/python2.7/site-packages to pythonpath.
WSGI app 0 (mountpoint='') ready in 0 seconds on interpreter 0x2465e40 pid: 8258 (default app)
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI worker 1 (and the only) (pid: 8258, cores: 1)
*** Starting uWSGI 2.0.10 (64bit) on [Sat May 23 15:25:51 2015] ***
compiled with version: 4.8.2 20140120 (Red Hat 4.8.2-16) on 23 May 2015 13:22:55
os: Linux-3.14.42-31.38.amzn1.x86_64 #1 SMP Wed May 13 20:33:05 UTC 2015
nodename: ip-172-31-21-203
machine: x86_64
clock source: unix
detected number of CPU cores: 1
current working directory: /home/ec2-user/projects/chaos.trade
detected binary path: /usr/local/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
chdir() to /home/ec2-user/projects/chaos.trade
*** WARNING: you are running uWSGI without its master process manager ***
your processes number limit is 7826
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address /home/ec2-user/projects/chaos.trade/chaos_uwsgi.sock fd 3
Python version: 2.7.9 (default, Apr  1 2015, 18:18:03)  [GCC 4.8.2 20140120 (Red Hat 4.8.2-16)]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x1943e40
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 72768 bytes (71 KB) for 1 cores
*** Operational MODE: single process ***
added /home/ec2-user/projects/chaos.trade:/usr/local/lib/python2.7/site-packages to pythonpath.
WSGI app 0 (mountpoint='') ready in 0 seconds on interpreter 0x1943e40 pid: 8263 (default app)
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI worker 1 (and the only) (pid: 8263, cores: 1)
*** Starting uWSGI 2.0.10 (64bit) on [Sat May 23 15:28:18 2015] ***
compiled with version: 4.8.2 20140120 (Red Hat 4.8.2-16) on 23 May 2015 13:22:55
os: Linux-3.14.42-31.38.amzn1.x86_64 #1 SMP Wed May 13 20:33:05 UTC 2015
nodename: ip-172-31-21-203
machine: x86_64
clock source: unix
detected number of CPU cores: 1
current working directory: /home/ec2-user/projects/chaos.trade
detected binary path: /usr/local/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
chdir() to /home/ec2-user/projects/chaos.trade
*** WARNING: you are running uWSGI without its master process manager ***
your processes number limit is 7826
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address /home/ec2-user/projects/chaos.trade/chaos_uwsgi.sock fd 3
Python version: 2.7.9 (default, Apr  1 2015, 18:18:03)  [GCC 4.8.2 20140120 (Red Hat 4.8.2-16)]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x1a5ae40
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 72768 bytes (71 KB) for 1 cores
*** Operational MODE: single process ***
added /home/ec2-user/projects/chaos.trade:/usr/local/lib/python2.7/site-packages to pythonpath.
WSGI app 0 (mountpoint='') ready in 0 seconds on interpreter 0x1a5ae40 pid: 8270 (default app)
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI worker 1 (and the only) (pid: 8270, cores: 1)
*** Starting uWSGI 2.0.10 (64bit) on [Sat May 23 15:34:18 2015] ***
compiled with version: 4.8.2 20140120 (Red Hat 4.8.2-16) on 23 May 2015 13:22:55
os: Linux-3.14.42-31.38.amzn1.x86_64 #1 SMP Wed May 13 20:33:05 UTC 2015
nodename: ip-172-31-21-203
machine: x86_64
clock source: unix
detected number of CPU cores: 1
current working directory: /home/ec2-user/projects/chaos.trade
detected binary path: /usr/local/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
chdir() to /home/ec2-user/projects/chaos.trade
*** WARNING: you are running uWSGI without its master process manager ***
your processes number limit is 7826
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address /home/ec2-user/projects/chaos.trade/chaos_uwsgi.sock fd 3
Python version: 2.7.9 (default, Apr  1 2015, 18:18:03)  [GCC 4.8.2 20140120 (Red Hat 4.8.2-16)]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x1623e40
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 72768 bytes (71 KB) for 1 cores
*** Operational MODE: single process ***
added /home/ec2-user/projects/chaos.trade:/usr/local/lib/python2.7/site-packages to pythonpath.
WSGI app 0 (mountpoint='') ready in 0 seconds on interpreter 0x1623e40 pid: 8281 (default app)
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI worker 1 (and the only) (pid: 8281, cores: 1)
*** Starting uWSGI 2.0.10 (64bit) on [Sat May 23 15:37:20 2015] ***
compiled with version: 4.8.2 20140120 (Red Hat 4.8.2-16) on 23 May 2015 13:22:55
os: Linux-3.14.42-31.38.amzn1.x86_64 #1 SMP Wed May 13 20:33:05 UTC 2015
nodename: ip-172-31-21-203
machine: x86_64
clock source: unix
detected number of CPU cores: 1
current working directory: /home/ec2-user/projects/chaos.trade
detected binary path: /usr/local/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
chdir() to /home/ec2-user/projects/chaos.trade
*** WARNING: you are running uWSGI without its master process manager ***
your processes number limit is 7826
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address /home/ec2-user/projects/chaos.trade/chaos_uwsgi.sock fd 3
Python version: 2.7.9 (default, Apr  1 2015, 18:18:03)  [GCC 4.8.2 20140120 (Red Hat 4.8.2-16)]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x1f96ec0
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 72768 bytes (71 KB) for 1 cores
*** Operational MODE: single process ***
added /home/ec2-user/projects/chaos.trade/ to pythonpath.
added /usr/local/lib/python2.7/site-packages/ to pythonpath.
WSGI app 0 (mountpoint='') ready in 0 seconds on interpreter 0x1f96ec0 pid: 8287 (default app)
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI worker 1 (and the only) (pid: 8287, cores: 1)
*** Starting uWSGI 2.0.10 (64bit) on [Sat May 23 15:45:45 2015] ***
compiled with version: 4.8.2 20140120 (Red Hat 4.8.2-16) on 23 May 2015 13:22:55
os: Linux-3.14.42-31.38.amzn1.x86_64 #1 SMP Wed May 13 20:33:05 UTC 2015
nodename: ip-172-31-21-203
machine: x86_64
clock source: unix
detected number of CPU cores: 1
current working directory: /home/ec2-user/projects/chaos.trade
detected binary path: /usr/local/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
chdir() to /home/ec2-user/projects/chaos.trade
your processes number limit is 7826
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address /home/ec2-user/projects/chaos.trade/chaos_uwsgi.sock fd 3
Python version: 2.7.9 (default, Apr  1 2015, 18:18:03)  [GCC 4.8.2 20140120 (Red Hat 4.8.2-16)]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x205df30
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 145536 bytes (142 KB) for 1 cores
*** Operational MODE: single process ***
added /home/ec2-user/projects/chaos.trade/ to pythonpath.
added /usr/local/lib/python2.7/site-packages/ to pythonpath.
WSGI app 0 (mountpoint='') ready in 0 seconds on interpreter 0x205df30 pid: 8356 (default app)
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 8356)
spawned uWSGI worker 1 (pid: 8359, cores: 1)
SIGINT/SIGQUIT received...killing workers...
worker 1 buried after 1 seconds
goodbye to uWSGI.
*** Starting uWSGI 2.0.10 (64bit) on [Sat May 23 15:53:11 2015] ***
compiled with version: 4.8.2 20140120 (Red Hat 4.8.2-16) on 23 May 2015 13:22:55
os: Linux-3.14.42-31.38.amzn1.x86_64 #1 SMP Wed May 13 20:33:05 UTC 2015
nodename: ip-172-31-21-203
machine: x86_64
clock source: unix
detected number of CPU cores: 1
current working directory: /home/ec2-user/projects/chaos.trade
detected binary path: /usr/local/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
chdir() to /home/ec2-user/projects/chaos.trade
your processes number limit is 7826
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address /home/ec2-user/projects/chaos.trade/chaos_uwsgi.sock fd 3
Python version: 2.7.9 (default, Apr  1 2015, 18:18:03)  [GCC 4.8.2 20140120 (Red Hat 4.8.2-16)]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x1f6bf30
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 145536 bytes (142 KB) for 1 cores
*** Operational MODE: single process ***
added /home/ec2-user/projects/chaos.trade/ to pythonpath.
added /usr/local/lib/python2.7/site-packages/ to pythonpath.
WSGI app 0 (mountpoint='') ready in 0 seconds on interpreter 0x1f6bf30 pid: 8397 (default app)
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 8397)
spawned uWSGI worker 1 (pid: 8400, cores: 1)
SIGINT/SIGQUIT received...killing workers...
worker 1 buried after 1 seconds
goodbye to uWSGI.
*** Starting uWSGI 2.0.10 (64bit) on [Sat May 23 15:58:32 2015] ***
compiled with version: 4.8.2 20140120 (Red Hat 4.8.2-16) on 23 May 2015 13:22:55
os: Linux-3.14.42-31.38.amzn1.x86_64 #1 SMP Wed May 13 20:33:05 UTC 2015
nodename: ip-172-31-21-203
machine: x86_64
clock source: unix
detected number of CPU cores: 1
current working directory: /home/ec2-user/projects/chaos.trade
detected binary path: /usr/local/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
chdir() to /home/ec2-user/projects/chaos.trade
your processes number limit is 7826
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address /home/ec2-user/projects/chaos.trade/chaos_uwsgi.sock fd 3
Python version: 2.7.9 (default, Apr  1 2015, 18:18:03)  [GCC 4.8.2 20140120 (Red Hat 4.8.2-16)]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0xc83f40
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 145536 bytes (142 KB) for 1 cores
*** Operational MODE: single process ***
added /home/ec2-user/projects/chaos.trade/ to pythonpath.
added /usr/local/lib/python2.7/site-packages/ to pythonpath.
WSGI app 0 (mountpoint='') ready in 0 seconds on interpreter 0xc83f40 pid: 8452 (default app)
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 8452)
spawned uWSGI worker 1 (pid: 8455, cores: 1)
SIGINT/SIGQUIT received...killing workers...
worker 1 buried after 1 seconds
goodbye to uWSGI.
*** Starting uWSGI 2.0.10 (64bit) on [Sat May 23 16:03:09 2015] ***
compiled with version: 4.8.2 20140120 (Red Hat 4.8.2-16) on 23 May 2015 13:22:55
os: Linux-3.14.42-31.38.amzn1.x86_64 #1 SMP Wed May 13 20:33:05 UTC 2015
nodename: ip-172-31-21-203
machine: x86_64
clock source: unix
detected number of CPU cores: 1
current working directory: /home/ec2-user/projects/chaos.trade
detected binary path: /usr/local/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
chdir() to /home/ec2-user/projects/chaos.trade
your processes number limit is 7826
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address /home/ec2-user/projects/chaos.trade/chaos_uwsgi.sock fd 3
Python version: 2.7.9 (default, Apr  1 2015, 18:18:03)  [GCC 4.8.2 20140120 (Red Hat 4.8.2-16)]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0xa28f40
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 145536 bytes (142 KB) for 1 cores
*** Operational MODE: single process ***
added /home/ec2-user/projects/chaos.trade/ to pythonpath.
added /usr/local/lib/python2.7/site-packages/ to pythonpath.
WSGI app 0 (mountpoint='') ready in 0 seconds on interpreter 0xa28f40 pid: 8492 (default app)
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 8492)
spawned uWSGI worker 1 (pid: 8495, cores: 1)
SIGINT/SIGQUIT received...killing workers...
worker 1 buried after 1 seconds
goodbye to uWSGI.
*** Starting uWSGI 2.0.10 (64bit) on [Sat May 23 16:08:33 2015] ***
compiled with version: 4.8.2 20140120 (Red Hat 4.8.2-16) on 23 May 2015 13:22:55
os: Linux-3.14.42-31.38.amzn1.x86_64 #1 SMP Wed May 13 20:33:05 UTC 2015
nodename: ip-172-31-21-203
machine: x86_64
clock source: unix
detected number of CPU cores: 1
current working directory: /home/ec2-user/projects/chaos.trade
detected binary path: /usr/local/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
chdir() to /home/ec2-user/projects/chaos.trade
your processes number limit is 7826
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address /home/ec2-user/projects/chaos.trade/chaos_uwsgi.sock fd 3
Python version: 2.7.9 (default, Apr  1 2015, 18:18:03)  [GCC 4.8.2 20140120 (Red Hat 4.8.2-16)]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0xcebf40
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 145536 bytes (142 KB) for 1 cores
*** Operational MODE: single process ***
added /home/ec2-user/projects/chaos.trade/ to pythonpath.
added /usr/local/lib/python2.7/site-packages/ to pythonpath.
unable to load app 0 (mountpoint='') (callable not found or import error)
*** no app loaded. going in full dynamic mode ***
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 8529)
spawned uWSGI worker 1 (pid: 8530, cores: 1)
SIGINT/SIGQUIT received...killing workers...
worker 1 buried after 0 seconds
goodbye to uWSGI.
*** Starting uWSGI 2.0.10 (64bit) on [Sat May 23 16:11:41 2015] ***
compiled with version: 4.8.2 20140120 (Red Hat 4.8.2-16) on 23 May 2015 13:22:55
os: Linux-3.14.42-31.38.amzn1.x86_64 #1 SMP Wed May 13 20:33:05 UTC 2015
nodename: ip-172-31-21-203
machine: x86_64
clock source: unix
detected number of CPU cores: 1
current working directory: /home/ec2-user/projects/chaos.trade
detected binary path: /usr/local/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
chdir() to /home/ec2-user/projects/chaos.trade
your processes number limit is 7826
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address /home/ec2-user/projects/chaos.trade/chaos_uwsgi.sock fd 3
Python version: 2.7.9 (default, Apr  1 2015, 18:18:03)  [GCC 4.8.2 20140120 (Red Hat 4.8.2-16)]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0xc52ed0
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 145536 bytes (142 KB) for 1 cores
*** Operational MODE: single process ***
added /home/ec2-user/projects/chaos.trade/ to pythonpath.
added /usr/local/lib/python2.7/site-packages/ to pythonpath.
unable to load app 0 (mountpoint='') (callable not found or import error)
*** no app loaded. going in full dynamic mode ***
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 8537)
spawned uWSGI worker 1 (pid: 8538, cores: 1)
SIGINT/SIGQUIT received...killing workers...
worker 1 buried after 1 seconds
goodbye to uWSGI.
*** Starting uWSGI 2.0.10 (64bit) on [Sat May 23 16:14:22 2015] ***
compiled with version: 4.8.2 20140120 (Red Hat 4.8.2-16) on 23 May 2015 13:22:55
os: Linux-3.14.42-31.38.amzn1.x86_64 #1 SMP Wed May 13 20:33:05 UTC 2015
nodename: ip-172-31-21-203
machine: x86_64
clock source: unix
detected number of CPU cores: 1
current working directory: /home/ec2-user/projects/chaos.trade
detected binary path: /usr/local/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
chdir() to /home/ec2-user/projects/chaos.trade
your processes number limit is 7826
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address /home/ec2-user/projects/chaos.trade/chaos_uwsgi.sock fd 3
Python version: 2.7.9 (default, Apr  1 2015, 18:18:03)  [GCC 4.8.2 20140120 (Red Hat 4.8.2-16)]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0xedaed0
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 145536 bytes (142 KB) for 1 cores
*** Operational MODE: single process ***
added /home/ec2-user/projects/chaos.trade/ to pythonpath.
added /usr/local/lib/python2.7/site-packages/ to pythonpath.
WSGI app 0 (mountpoint='') ready in 0 seconds on interpreter 0xedaed0 pid: 8552 (default app)
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 8552)
spawned uWSGI worker 1 (pid: 8555, cores: 1)
SIGINT/SIGQUIT received...killing workers...
worker 1 buried after 1 seconds
goodbye to uWSGI.
*** Starting uWSGI 2.0.10 (64bit) on [Sat May 23 16:20:20 2015] ***
compiled with version: 4.8.2 20140120 (Red Hat 4.8.2-16) on 23 May 2015 13:22:55
os: Linux-3.14.42-31.38.amzn1.x86_64 #1 SMP Wed May 13 20:33:05 UTC 2015
nodename: ip-172-31-21-203
machine: x86_64
clock source: unix
detected number of CPU cores: 1
current working directory: /home/ec2-user/projects/chaos.trade
detected binary path: /usr/local/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
your processes number limit is 7826
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address /home/ec2-user/projects/chaos.trade/chaos_uwsgi.sock fd 3
Python version: 2.7.9 (default, Apr  1 2015, 18:18:03)  [GCC 4.8.2 20140120 (Red Hat 4.8.2-16)]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x1180df0
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 145536 bytes (142 KB) for 1 cores
*** Operational MODE: single process ***
added /home/ec2-user/projects/chaos.trade/ to pythonpath.
added /usr/local/lib/python2.7/site-packages/ to pythonpath.
*** no app loaded. going in full dynamic mode ***
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 8565)
spawned uWSGI worker 1 (pid: 8566, cores: 1)
SIGINT/SIGQUIT received...killing workers...
worker 1 buried after 1 seconds
goodbye to uWSGI.
*** Starting uWSGI 2.0.10 (64bit) on [Sat May 23 16:28:01 2015] ***
compiled with version: 4.8.2 20140120 (Red Hat 4.8.2-16) on 23 May 2015 13:22:55
os: Linux-3.14.42-31.38.amzn1.x86_64 #1 SMP Wed May 13 20:33:05 UTC 2015
nodename: ip-172-31-21-203
machine: x86_64
clock source: unix
detected number of CPU cores: 1
current working directory: /home/ec2-user/projects/chaos.trade
detected binary path: /usr/local/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
your processes number limit is 7826
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to TCP address 127.0.0.0:5000 fd 3
Python version: 2.7.9 (default, Apr  1 2015, 18:18:03)  [GCC 4.8.2 20140120 (Red Hat 4.8.2-16)]
Python main interpreter initialized at 0x9be1a0
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 415360 bytes (405 KB) for 8 cores
*** Operational MODE: preforking+threaded ***
added /home/ec2-user/projects/chaos.trade/ to pythonpath.
added /usr/local/lib/python2.7/site-packages/ to pythonpath.
*** no app loaded. going in full dynamic mode ***
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 8611)
spawned uWSGI worker 1 (pid: 8612, cores: 2)
spawned uWSGI worker 2 (pid: 8613, cores: 2)
spawned uWSGI worker 3 (pid: 8614, cores: 2)
spawned uWSGI worker 4 (pid: 8615, cores: 2)
...brutally killing workers...
worker 1 buried after 1 seconds
worker 2 buried after 1 seconds
worker 3 buried after 1 seconds
worker 4 buried after 1 seconds
binary reloading uWSGI...
chdir() to /home/ec2-user/projects/chaos.trade
closing all non-uwsgi socket fds > 2 (max_fd = 1024)...
found fd 3 mapped to socket 0 (127.0.0.0:5000)
running /usr/local/bin/uwsgi
[uWSGI] getting INI configuration from chaos_uwsgi.ini
*** Starting uWSGI 2.0.10 (64bit) on [Sat May 23 16:29:02 2015] ***
compiled with version: 4.8.2 20140120 (Red Hat 4.8.2-16) on 23 May 2015 13:22:55
os: Linux-3.14.42-31.38.amzn1.x86_64 #1 SMP Wed May 13 20:33:05 UTC 2015
nodename: ip-172-31-21-203
machine: x86_64
clock source: unix
detected number of CPU cores: 1
current working directory: /home/ec2-user/projects/chaos.trade
detected binary path: /usr/local/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
your processes number limit is 7826
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 inherited INET address 127.0.0.0:5000 fd 3
Python version: 2.7.9 (default, Apr  1 2015, 18:18:03)  [GCC 4.8.2 20140120 (Red Hat 4.8.2-16)]
Python main interpreter initialized at 0x1402c80
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 415360 bytes (405 KB) for 8 cores
*** Operational MODE: preforking+threaded ***
added /home/ec2-user/projects/chaos.trade/ to pythonpath.
added /usr/local/lib/python2.7/site-packages/ to pythonpath.
*** no app loaded. going in full dynamic mode ***
*** uWSGI is running in multiple interpreter mode ***
gracefully (RE)spawned uWSGI master process (pid: 8611)
spawned uWSGI worker 1 (pid: 8623, cores: 2)
spawned uWSGI worker 2 (pid: 8624, cores: 2)
spawned uWSGI worker 3 (pid: 8625, cores: 2)
spawned uWSGI worker 4 (pid: 8626, cores: 2)
Sat May 23 16:29:26 2015 - uWSGI worker 1 screams: UAAAAAAH my master disconnected: i will kill myself !!!
Sat May 23 16:29:26 2015 - uWSGI worker 2 screams: UAAAAAAH my master disconnected: i will kill myself !!!
Sat May 23 16:29:26 2015 - uWSGI worker 3 screams: UAAAAAAH my master disconnected: i will kill myself !!!
Sat May 23 16:29:26 2015 - uWSGI worker 4 screams: UAAAAAAH my master disconnected: i will kill myself !!!
